Elasticsearch Devservices (#23753)
quarkusio/quarkus: docs/src/main/asciidoc/hibernate-search-orm-elasticsearch.adoc

[TIP]
For more information about the Hibernate Search extension configuration please refer to the <<configuration-reference, Configuration Reference>>.

[[dev-services]]
=== Dev Services (Configuration Free Databases)

> Quarkus supports a feature called Dev Services that allows you to create various datasources without any config.

Reviewer: ```suggestion
Quarkus supports a feature called Dev Services that allows you to start various containers without any config.
```

In the case of Elasticsearch this support extends to the default Elasticsearch connection.
What that means practically, is that if you have not configured `quarkus.hibernate-search-orm.elasticsearch.hosts` Quarkus will automatically

> start an Elasticsearch container when running tests or dev-mode, and automatically configure the connection.

Reviewer: ```suggestion
start an Elasticsearch container when running tests or dev mode, and automatically configure the connection.
```

> When running the production version of the application, the Elasticsearch connection need to be configured as normal,

Reviewer: ```suggestion
When running the production version of the application, the Elasticsearch connection needs to be configured as normal,
```
